connexion/decorators/parameter.py

```diff
@@ -77,14 +83,14 @@ def wrapper(*args, **kwargs):
                 kwargs[body_name] = request_body
 
         # Add query parameters
-        for key, value in flask.request.args.items():
+        for key, value in list(flask.request.args.items()) + list(default_query_params.items()):
```

Comment:
When I looked at it more closely I saw there is a problem with which value is used. I have fixed this issue by using a python `dict` and then `update()`ing it. Seemed a better solution on the whole.